book/partner-resources/prep-for-kickoff.md

```diff
@@ -0,0 +1,35 @@
+# Getting Started with Building a Public Plugin
+### Partner Documentation
+
+### Purpose
+Public plugins are presented to all customers in the marketplace and are openly available for use. This document outlines the initial steps of preparing for developing a public plugin and preparing for the development kickoff call with marketplace that will help get things started.
+
+### Checklist
+
+- If you haven’t established a relationship with our Business Development team yet, reach out to Erik Koland (e.kolad@applicaster.com) in order to do so. This can help establish a mutually beneficial relationship so you can best capitalize on the opportunities our marketplace enables.
+
+- Prep for your plugin kickoff call with the Marketplace Team. This will be a good time to raise any open questions or request feedback. You should check off the following items prior to the call:
+	- Identify your full product scope for your plugin so we can guide you on choices for which Zapp APIs are best suited to your needs
+	- Create a request for starting [the onboarding process for your developers](https://applicaster.atlassian.net/servicedesk/customer/portal/2/group/3/create/18) (if it is the first time, click the “signup” button) 
+	- Identify who will fill the roles of Product Manager and Project Manager for your plugin development. 
+	- Reach out to Yoni Osteen from the marketplace team (y.osteen@applicaster.com) to get a development kickoff call set up
```

Comment:
Your team needs a special email. 
You can talk with Liat about that.

Reply:
I am the only person in my role. It is not a team. Sending it to a marketplace team generic email is not right as who will review that inbox? Why create such an email address if it is limited to partnership product success content? Creating a team email for one person seems like unnecessary bureaucracy and also looks kind of disingenuous when a partner realizes that it is basically a second email for me.  Let's hop on a call if you feel otherwise. 

Fwiw, I had requested using a scheduling software that interfaces with Google Calendar and was hoping to link to that here, but was told we don't have budget for new software. I've checked if MS has an existing account with something, but apparently they don't. Will look to see if there is a good free software, but am skeptical.